src/apps/loan-list/modal/material-details.tsx

```diff
@@ -54,7 +59,21 @@ const MaterialDetails: FC<MaterialDetailsProps & MaterialProps> = ({
           />
         )}
       </ModalDetailsHeader>
-      {faust && <RenewButton faust={faust} renewable={isRenewable} />}
+      {!isDigital(loan) && faust && (
+        <RenewButton faust={faust} renewable={isRenewable} />
+      )}
+      {isDigital(loan) && (
+        <div className="modal-details__buttons">
+          {/* todo create a component for ereolen-redirect (also replace url) */}
+          <Link
+            href={new URL("https://ereolen.dk/user/me/")}
```

Comment:
Should we make this a data prop url so it can be configured by the cms?

Reply:
{/* todo create a component for ereolen-redirect (also replace url) */} <-